src/OnlineRetailPortal.Contracts/Contracts/IProductStoreFactory.cs

```diff
@@ -6,5 +6,6 @@ namespace OnlineRetailPortal.Contracts
 {
     public interface IProductStoreFactory
     {
+        IProductStore GetStoreType(string storeValue);
```

Comment:
Please make sure this class is use somewhere if not then remove

Reply:
used in service class. The code is in another pr